examples/bravais_lattice/fcc_SiO2/INPUT

```diff
@@ -3,8 +3,13 @@ INPUT_PARAMETERS
 pseudo_dir              ../../../tests/PP_ORB	
 calculation		scf
 #Parameters (Accuracy)
-ecutwfc			50
+ecutwfc			 60 ###Energy cutoff needs to be tested to ensure your calculation is reliable.[1]
 scf_thr         	1e-8
 scf_nmax		20
 basis_type		pw
 latname			fcc
+
+
+### [1] Energy cutoff determines the quality of numerical quadratures in your calculations.
```

Comment:
we only suggest testing the ecut for plane wave basis set, for lcao basis set, we recommend not testting it but just use the cutoff in LCAO basis sets.

Reply:
This is done and I made a new commit. Could you please help me double check it?

Reply:
OK